commands/schema/filecoin_block.schema.json

```diff
@@ -52,6 +52,12 @@
         },
         "timestamp": {
           "type": "string"
+        },
```

Comment:
I know this is going away soon but it was easier to add this than remove the whole thing to get CI green

Reply:
Please do the honours for us in a different PR